test/versions.bats

```diff
@@ -116,14 +116,52 @@ OUT
 OUT
 }
 
+@test "indicates current version when using lts/<codename> alias" {
+  stub_system_node
+  create_version "1.9.3"
+  create_version "2.0.0"
+  mkdir "${NODENV_ROOT}/versions/lts"
+  ln -s "../1.9.3" "${NODENV_ROOT}/versions/lts/bar"
+
+  NODENV_VERSION=lts/bar run nodenv-versions
+
+  assert_success
+  assert_output - <<OUT
+  system
+  1.9.3
+  2.0.0
+* lts/bar => 1.9.3 (set by NODENV_VERSION environment variable)
+OUT
+}
+
+@test "indicates current version when using lts as a direct alias" {
```

Comment:
Also happy about better ideas for the description here, I was struggling to find something that accurately describes the difference here 😅